app/controllers/medicaid/health_pregnancy_controller.rb

```diff
@@ -2,5 +2,14 @@
 
 module Medicaid
   class HealthPregnancyController < MedicaidStepsController
+    private
+
+    def skip?
+      client_is_male?
+    end
+
+    def client_is_male?
+      current_application.primary_member.sex == "Male"
```

Comment:
what if there are multiple members and one is female?

Reply:
Great idea